<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TAudio</class>
 <widget class="QWidget" name="Gui::Pref::TAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>576</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="audio_tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="audio_tab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="margin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="output_layout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="icon_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="output_group">
           <property name="title">
            <string>Output</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="ao_label">
                <property name="text">
                 <string>&amp;Driver:</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="buddy">
                 <cstring>ao_combo</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="ao_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ao_user_defined_edit"/>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>81</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="channels_label">
                <property name="text">
                 <string>&amp;Channels:</string>
                </property>
                <property name="buddy">
                 <cstring>channels_combo</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="channels_combo"/>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="hwac3_check">
              <property name="text">
               <string>&amp;AC3/DTS pass-through S/PDIF</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="audio_equalizer_check">
              <property name="text">
               <string>&amp;Enable the audio equalizer</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="scaletempo_label">
                <property name="text">
                 <string>&amp;High speed playback without altering pitch</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="buddy">
                 <cstring>scaletempo_combo</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Gui::Pref::TTristateCombo" name="scaletempo_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="volume_group">
         <property name="title">
          <string>Volume</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="softvol_check">
              <property name="text">
               <string>&amp;Use software volume control</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="amplification_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>A&amp;mplification:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>softvol_amp_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="softvol_amp_spin">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>110</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="volnorm_check">
            <property name="text">
             <string>Volume &amp;normalization by default</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="synchronization_group">
         <property name="title">
          <string>Synchronization</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="autosync_check">
              <property name="text">
               <string>Audio/video auto &amp;synchronization</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="factor_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>&amp;Factor:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>autosync_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autosync_spin">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="use_mc_check">
              <property name="text">
               <string>A-V sync &amp;correction</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="mc_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>&amp;Max. correction:</string>
              </property>
              <property name="buddy">
               <cstring>mc_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mc_spin">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="language_group">
         <property name="title">
          <string>Language</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="language_label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Override the audio channel selected by the player with &lt;span style=&quot; font-weight:600;&quot;&gt;language&lt;/span&gt;. This field accepts regular expressions.&lt;br/&gt;Example: &lt;span style=&quot; font-weight:600;&quot;&gt;es|esp|spa&lt;/span&gt; selects the first audio track with a language matching &lt;span style=&quot; font-style:italic;&quot;&gt;es&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;esp&lt;/span&gt; or &lt;span style=&quot; font-style:italic;&quot;&gt;spa&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>language_edit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="language_edit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Pref::TTristateCombo</class>
   <extends>QComboBox</extends>
   <header>gui/pref/tristatecombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>audio_tab_widget</tabstop>
  <tabstop>ao_combo</tabstop>
  <tabstop>ao_user_defined_edit</tabstop>
  <tabstop>channels_combo</tabstop>
  <tabstop>hwac3_check</tabstop>
  <tabstop>audio_equalizer_check</tabstop>
  <tabstop>scaletempo_combo</tabstop>
  <tabstop>softvol_check</tabstop>
  <tabstop>softvol_amp_spin</tabstop>
  <tabstop>volnorm_check</tabstop>
  <tabstop>autosync_check</tabstop>
  <tabstop>autosync_spin</tabstop>
  <tabstop>use_mc_check</tabstop>
  <tabstop>mc_spin</tabstop>
  <tabstop>language_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>softvol_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>softvol_amp_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>softvol_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>amplification_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosync_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>autosync_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosync_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>factor_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_mc_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>mc_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_mc_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>mc_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
